rust_src/src/fonts.rs

```diff
@@ -117,7 +117,7 @@ impl LispFontObjectRef {
         unsafe { font_add_log(c_str.as_ptr(), self.into(), result.into()) }
     }
 
```

Comment:
`_frame` was used here because when `window-system` is not enabled the `frame` argument is unused and clippy rightly complains.

Reply:
Clippy was complaining about that when I ran it locally, but I guess it just doesn't handle this situation very well.